service/crawlingKeywordService.js

```diff
@@ -1,25 +1,81 @@
 const puppeteer = require("puppeteer");
 
+function isCheckTrueThisUrl(url) {
+  /* eslint-disable */
+  const urlRegex = /^(http|https):\/\/[\w\-]+(\.[\w\-]+)+[/#?]?.*$/;
+  if (urlRegex.test(url)) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
 const getCrawlingKeyword = async (req, res) => {
-  const decodedLink = decodeURIComponent(req.params.url);
+  const decodedUrl = decodeURIComponent(req.params.url);
   const browser = await puppeteer.launch({ headless: true });
+  const TIMEOUT = 20000;
 
   try {
     const page = await browser.newPage();
 
-    await page.goto(decodedLink);
+    await page.goto(decodedUrl);
 
     let innerText = await page.evaluate(() => document.body.innerText);
     const hasKeyword = innerText.includes(req.query.keyword);
 
-    return res.status(200).json({
-      urlLink: req.params.url,
-      keyword: req.query.keyword,
-      hasKeyword: hasKeyword,
-      urlText: innerText,
-    });
+    if (hasKeyword === true) {
```

Comment:
```suggestion
    if (hasKeyword) {
```
이 부분은 이런 식으로도 변형이 가능할 것으로 보입니다.
다음은 주로 사용되는 AirBnB에서 정의한 조건에 대한 [좋은 컨벤션](https://github.com/tipjs/javascript-style-guide?tab=readme-ov-file#15.3) 입니다.